configs/example/kmhv3.py

```
@@ -126,6 +126,9 @@ def setKmhV3Params(args, system):
                 system.l2_caches[i].wpu = NULL
                 system.l2_caches[i].do_fast_writeline = False
                 system.l2_caches[i].prefetch_can_offload = False
+                # Configure XSDRRIP replacement policy (DRRIP mode)
+                # L2: 2MB, 8-way, 64B line → 4096 sets
+                system.l2_caches[i].replacement_policy = XSDRRIPRP(mode=2, num_sets=4096)
```

Ruff (0.14.10)

`XSDRRIPRP` may be undefined, or defined from star imports

(F405)